src/test/java/uk/gov/dhsc/htbhf/smartstub/helper/PersonTestFactory.java

```diff
@@ -0,0 +1,30 @@
+package uk.gov.dhsc.htbhf.smartstub.helper;
+
+import uk.gov.dhsc.htbhf.smartstub.model.PersonDTO;
+
+import static uk.gov.dhsc.htbhf.smartstub.helper.AddressDTOTestDataFactory.aValidAddress;
+import static uk.gov.dhsc.htbhf.smartstub.helper.TestConstants.HOMER_DATE_OF_BIRTH;
+import static uk.gov.dhsc.htbhf.smartstub.helper.TestConstants.HOMER_FIRST_NAME;
+import static uk.gov.dhsc.htbhf.smartstub.helper.TestConstants.SIMPSON_LAST_NAME;
+
+/**
+ * Test data factory for {@link PersonDTO} objects.
+ */
+public class PersonTestFactory {
+
+    private static final String NINO = "EB123456C";
+
+    public static PersonDTO aValidPerson() {
+        return buildDefaultPerson().build();
+    }
+
+    public static PersonDTO.PersonDTOBuilder buildDefaultPerson() {
```

Comment:
- Address DTO: `aValidAddress` and `aValidAddressBuilder`
- Person DTO: `aValidPerson` and  `buildDefaultPerson`

Any reason for the difference in naming. If possible would be good to keep consistent between test data factories.

Reply:
No reason, it was copy and pasted from smart stub. I've renamed buildDefaultPerson to aValidPersonBuilder.